Missing dependency 'bunyan'

``` javascript
var bunyan = require('bunyan');
module.exports = bunyan.createLogger({
  name: 'biodome-client',
  stream: process.stdout
});
```

biodome/lib/log.js:1:76
Error: Cannot find module 'bunyan'

--- a/package.json
+++ b/package.json
@@ -26,7 +26,8 @@
     "chai": "1.8.x",
     "mocha": "1.14.x",
     "sinon": "1.7.x",
-    "supertest": "0.8.x"
+    "supertest": "0.8.x",
+    "bunyan": "~0.22.1"
   },
   "repository": {
     "type": "git",

